### ISSUE TITLE
how can run 

### ISSUE THREAD

**Issue by rokmc612** [krasserm/super-resolution#43]
Hello!!

i try to Xilinx alveo card used by user code.  but, Xilinx was recommended to run with  inference mode not training mode.   So, I modified to set to tf.keras.backend.set_learning_phase(1) into the  ./optimizer/weightnorm.py.   is it right?

would you pls advice to set of  inference mode runing for your code.

the below is Xilinx msg
=========================================================================

check that “is_training” flag is set to False before freezing the network? 

 

How to get the frozen inference graph?
    Frozen inference graph:
        Variables are frozen to Constants
        Graph and weights are in one single file
    Only inference phase graph should be kept to make the graph clear for pattern matching
        Set phase to eval/inference the graph before freezing
        For tf.layers.batchnorm/dropout, set is_training to False
        For tf.keras.layers.BatchNorm/Dropout, call  tf.keras.backend.set_learning_phase(0) # 0 for test, 1 for train
